lib/pages/settings/view/settings_page.dart

```diff
@@ -114,6 +115,34 @@ class _SettingsPageState extends State<SettingsPage> {
                     S.current.infoReadThePolicy(Utils.packageInfo.appName),
                   ),
                 ),
+                Visibility(
+                  child: PreferenceTitle(S.current.settingsTitleTimetable),
+                  visible: Platform.isAndroid || Platform.isIOS,
+                ),
+                Visibility(
+                  child: ListTile(
+                    key: const ValueKey('google_calendar'),
+                    onTap: () async {
+                      if (authProvider.isAnonymous) {
+                        AppToast.show(S.current.messageNotLoggedIn);
+                      } else if (isVerified != true) {
+                        AppToast.show(
+                            S.current.messageEmailNotVerifiedToPerformAction);
+                      } else {
+                        final eventProvider = Provider.of<UniEventProvider>(
+                            context,
+                            listen: false);
+                        await eventProvider.exportToGoogleCalendar();
+                      }
+                    },
+                    enabled: Provider.of<AuthProvider>(context, listen: false)
+                            .currentUserFromCache !=
+                        null,
+                    title: Text(S.current.settingsExportToGoogleCalendar),
+                    subtitle: Text(S.current.infoExportToGoogleCalendar),
+                  ),
+                  visible: Platform.isAndroid || Platform.isIOS,
```

Comment:
Nit: The child should usually be the last defined parameter in a widget. Please move visible: before it (here and above).